cli/src/stake.rs

```diff
@@ -1512,9 +1513,17 @@ pub fn build_stake_state(
             stake,
         ) => {
             let current_epoch = clock.epoch;
-            let (active_stake, activating_stake, deactivating_stake) = stake
-                .delegation
-                .stake_activating_and_deactivating(current_epoch, Some(stake_history));
+            let (active_stake, activating_stake, deactivating_stake) = match stake_activation.state
+            {
+                StakeActivationState::Active => (stake_activation.active, 0, 0),
+                StakeActivationState::Inactive => (0, 0, 0),
+                StakeActivationState::Activating => {
+                    (stake_activation.active, stake_activation.inactive, 0)
+                }
+                StakeActivationState::Deactivating => {
+                    (stake_activation.active, 0, stake_activation.inactive)
+                }
```

Comment:
shameless inverse conversion from RpcStakeActivation back to the original stake tuple (`(u64, u64, u64)`). If I had more time, I would fix CliStakeState itself... ;)